src/components/header/styleHeader/searchField.scss

```diff
@@ -0,0 +1,33 @@
+input {
+    position: relative;
+    display: inline-block;
+    font-size: 20px;
+    box-sizing: border-box;
+    transition: all 0.5s;
+
+    &[type="text"] {
+        background: #fff;
+        width: 350px;
+        height: 40px;
+        border: none;
+        outline: none;
+        padding: 0 25px;
+        border-radius: 25px 0 0 25px;
+    }
+
+    &[type="submit"] {
+        position: relative;
+        border-radius: 0 25px 25px 0;
+        height: 40px;
+        width: 100px;
+        border: none;
+        outline: none;
+        cursor: pointer;
+        background: rgb(241, 160, 8);
```

Comment:
This color is set as a primary color inside in header.scss. Please move it in one place and use it all over the app.